### PR TITLE
Chrome open

### PR DESCRIPTION
**Pull request by mitya52**

### CODE REVIEW

File: src/integrations/integr_chrome.rs [hunk 1]
```diff
@@ -39,9 +48,49 @@ pub struct ToolChrome {
...
+    }
+}
+
+#[derive(Clone)]
+pub struct ChromeTab {
+    headless_tab: Arc<HeadlessTab>,
+    device: DeviceType,
+    tab_id: String,
+    screenshot_scale_factor: f64,
+    // NOTE: logs vector should be at least limited
```
Comment (olegklimov) [inline src/integrations/integr_chrome.rs:72]: Use output_filter

File: src/integrations/integr_chrome.rs [hunk 2]
```diff
@@ -224,141 +289,325 @@ async fn setup_chrome_session(
...
+            };
+            let log = {
+                let tab_lock = tab.lock().await;
+                match {
+                    let mapped_point = Point {
+                        x: args.point.x / tab_lock.screenshot_scale_factor,
+                        y: args.point.y / tab_lock.screenshot_scale_factor,
+                    };
+                    tab_lock.headless_tab.click_point(mapped_point).map_err(|e| e.to_string())?;
+                    tab_lock.headless_tab.wait_until_navigated().map_err(|e| e.to_string())?;
```
Comment (olegklimov) [inline src/integrations/integr_chrome.rs:532]: will exit the whole function

**Review by olegklimov — APPROVED** [review]

File: src/integrations/integr_chrome.rs (hunk 2 above)
Comment (mitya52) [reply src/integrations/integr_chrome.rs:538]: tested, works well
it stores an error in the tool_log, all ok

File: src/integrations/integr_chrome.rs (hunk 1 above)
Comment (mitya52) [reply src/integrations/integr_chrome.rs]: added filter

**Review by olegklimov — APPROVED** [review]

**Pull request merged**